cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_controller.go

```diff
@@ -274,33 +277,66 @@ func (c *machineController) machinesInMachineSet(machineSet *MachineSet) ([]*Mac
 	return result, nil
 }
 
+// getMachineAPIGroup returns a string that specifies the version and
+// group for the API. It will return either the value from the
+// MACHINEAPI_GROUP environment variable, or the default
+// value. This value will be a string in the format of "version.group".
```

Comment:
Is it `version.group`? Cause the default is just the group and doesn't include the version 🤔 Looking at the rest of the code I'm suspecting the comment is wrong

Reply:
yeah, this comment needs to be updated